assets/js/api.js

```diff
@@ -27,8 +32,6 @@ function callStackAPI() {
     }).then(function (response) {
         for (var i = 0; i < 5; i++) {
             var qstnObject = {};
```

Comment:
var keywordURL = "";	
            var keywordTitle = "";
is required at this line..to make the function work properly